src/NexusMods.StandardGameLocators/Services.cs

```diff
@@ -112,6 +116,13 @@ public static class Services
                         ),
                     }
                 ));
+            },
+        onOSX: () =>
+            {
+    #pragma warning disable CA1416
+                services.AddSingleton<IHardwareInfoProvider, HardwareInfoProvider>();
+    #pragma warning restore CA1416
```

Comment:
`IHardwareInfoProvider` is required for EA Desktop, and the `HardwareInfoProvider` implementation is Windows-only.

This should be removed.